kraken-sdk/Cargo.toml

```diff
@@ -16,6 +16,7 @@ serde_json = "~1"
 url = "~2"
 uuid = { version = "~1", features = ["serde", "v4"] }
 ipnetwork = { version = "~0.20", features = ["serde"] }
+chrono = { version = "~0.4", default-features = false, features = ["serde"] }
```

Comment:
Change to version = ">=0.4.20", see https://docs.rs/chrono/latest/chrono/#security-advisories